client/my-sites/people/role-select/index.jsx

```diff
@@ -135,5 +137,19 @@ export default React.createClass( {
 				}
 			</FormFieldset>
 		);
+	},
+
+	renderSelect() {
+		const options = [ this.props.defaultOption ];
+		forEach( this.state.roles, ( roleObject, key ) => {
+			options.push( { value: key, label: roleObject.display_name, path: this.props.calculatePath(key) } );
+		} );
+		return (
+			<SelectDropdown compact options={options} onSelect={ this.props.onSelect } />
+		);
+	},
+
+	render() {
+		return this.props.fieldset ? this.renderFormFieldset() : this.renderSelect();
```

Comment:
It doesn’t look like we have documented the new `prop` anywhere? Also did the default behavior change?